tests-functional/tests/test_firstrun.py

```diff
@@ -0,0 +1,68 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.firstrun import Firstrun
+
+
+@pytest.mark.nondestructive
+def test_open_firstrun_page(base_url, selenium):
+    page = Firstrun(selenium, base_url).open()
+    # ensure all-aboard dialog heading text match
+    assert 'Have you used Firefox in the last 30 days?' == page.question_one_heading  # noqa E501
+    # ensure Firefox accounts form is not visible
+    assert page.fxa_form_visible() == False
+
+
+@pytest.mark.nondestructive
+def test_firstrun_allaboard_dialog_interaction(base_url, selenium):
+    page = Firstrun(selenium, base_url).open()
+    # confirms that the second part of the survey is not initially visible
+    assert page.what_matters_visible() == False
+    # ensure the Go! button is not visible
+    assert page.submit_button_visible() == False
+    # ensure that the no thanks link is visible
+    assert page.dismiss_visible() == True
```

Comment:
You don't need to compare this to `True`, instead you can do `assert page.dismiss_visible()`.